python/chemiscope/explore.py

```diff
@@ -163,3 +173,194 @@ def soap_pca_featurize(frames):
     # Compute pca
     pca = PCA(n_components=2)
     return pca.fit_transform(feats)
+
+
+def torchscript_featurizer(model, extensions_directory="extensions/", device=None):
+    """
+    Create a featurizer function using a `metatrain`_  model.
```

Comment:
So instead of just metatrain models, I think this should be able to take arbitrary metatensor models (metatrain is one way to create such model, but there are other ways).

Reply:
Since I primarily use the [MetatensorCalculator](https://docs.metatensor.org/latest/atomistic/reference/ase.html#metatensor.torch.atomistic.ase_calculator.MetatensorCalculator), I believe it takes care of this. I will change the docstring